documentation/resource_machine_name.md

```diff
@@ -0,0 +1,88 @@
+machine_name
+============
+
+Use the **machine_name** resource to manage a machine's name.
+
+As defined by the `scutil` manual, an individual macOS system has three different
+types of names managed by `scutil`: `ComputerName`, LocalHostName, and HostName.
```

Comment:
Add back ticks before `LocalHostName` and `HostName` to align with `ComputerName`.

Reply:
I'm actually not sure ComputerName should have them.

Reply:
If that's the case, we should remove them from line 14 and 15 for consistency.